### PR TITLE
Update to work against snap master and remove external mocking framework

### PR DESCRIPTION
**Pull request by mkleina**
Plugin was updated to work against snap v0.13.0-beta-102-g129af14. Additionally, according to new recommendations, external mocking frameworks was removed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,216 +135,216 @@ See file output (this is just part of the file):
...
+/intel/osv/trace/vfs/vfs_futimens| uint64|
+/intel/osv/trace/vfs/vfs_futimens_ret| uint64|
+/intel/osv/trace/vfs/vfs_futimens_err| uint64|
+/intel/osv/trace/vfs/vfs_chmod| uint64|
+/intel/osv/trace/vfs/vfs_chmod_ret| uint64|
+/intel/osv/trace/vfs/vfs_chmod_err| uint64|
+/intel/osv/trace/vfs/vfs_fchmod| uint64|
+/intel/osv/trace/vfs/vfs_fchmod_ret| uint64|
+/intel/osv/trace/vfs/vfs_fchown| uint64|
+/intel/osv/trace/vfs/vfs_fchown_ret| uint64|
```
Comment (marcintao) [inline README.md]: The list of metrics is long, making it difficult to read all of the README. Consider moving to external document, METRICS.MD, see [meminfo collector](https://github.com/intelsdi-x/snap-plugin-collector-meminfo#collected-metrics) for example.

Comment (sandlbn) [inline README.md]: @marcintao I can move it in next PR because I will need to expose more metrics requested by our partners.

**Comment by sandlbn** [comment]
LGTM @mkleina can you rebase ?

**Comment by mkleina** [comment]
@sandlbn It is already rebased to Intel master

**Pull request merged**